multiview/embed/omnibus.py

```diff
@@ -0,0 +1,160 @@
+"""
+omnibus.py
+==========
+Omnibus embedding for multiview dimensionality reduction.
+Code from the https://github.com/neurodata/graspy package,
+reproduced and shared with permission.
+"""
+
+from .base import BaseEmbed
+from ..utils.utils import check_Xs
+
+import numpy as np
+from graspy.embed import OmnibusEmbed
+from sklearn.metrics import pairwise_distances
+from sklearn.preprocessing import normalize
+
+
+class Omnibus(BaseEmbed):
+    """
+    Omnibus computes the pairwise distances for each view. Each
+    of these matrices is a n x n dissimilarity matrix where n is the number
+    of rows in each view. Omnibus embedding
+    (https://graspy.neurodata.io/reference/embed.html#multiple-graph-embedding)
+    is then performed over the dissimilarity matrices and the
+    computed embeddings are returned.
+
+    Parameters
+    ----------
+    n_components : strictly positive int (default = 2)
+        Desired dimensionality of output embeddings. See graspy docs for
+        additional details.
+
+    distance_metric : string (default = 'euclidean')
+        Distance metric used to compute pairwise distances.
+
+    normalize : string or None (default = 'l1')
+        Normalize function to use on views before computing
+        pairwise distances. Must be 'l2', 'l1', 'max'
+        or None. If None, the distance matrices will not be normalized.
+
+    algorithm : string (default = 'randomized')
+        SVD solver to use. Must be 'full', 'randomized', or 'truncated'.
+        See graspy docs for details.
```

Comment:
[note for future] you can set up something called intersphinx mapping that will automatically make this into a link to the most up to date version of graspy docs

Reply:
Good to know. Can set up an issue right now